recogni/zephyr
# Fix syslog bugs

Ported over these commits from mainline to fix syslog 64 bit & RISC-V
issues:
cda619996e3e0e710bdb
dc4e331e45cb1dc70b1

To Test:
---------------
On host linux machine:
Uncomment these lines in /etc/rsyslog.conf:
   module(load="imudp")
   input(type="imudp" port="514")
$ sudo systemctl restart rsyslog
$ tail -f /var/log/syslog

Make sure IP addresses in prj.conf are correct, then...
$ rm -rf build/
$ west build -b genesys2 samples/net/syslog_net/

When app runs, messages should appear in var/log/syslog on remote
machine.
